### DISCUSSION TITLE
Restrict access to Nexcloud, but leave access to download published files from Internet

### DISCUSSION THREAD

**Discussion by lexhatred** [nextcloud/all-in-one#2675] [Questions] [answered]
Hello! How can I restrict access to Nexcloud only for local network users, but leave access to download published files via links for the entire Internet?

**Comment by szaimen** [comment, marked as answer]
Will be answered in https://help.nextcloud.com/t/restrict-access-to-nexcloud-but-leave-access-to-download-published-files-from-internet/163476
